bundleresolver/config/bundleresolver-config.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright 2022 The Tekton Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: bundleresolver-config
+  namespace: tekton-remote-resolution
+data:
+  # The maximum amount of time a single git resolution may take.
+  default-service-account: "https://github.com/sbwsg/catalog.git"
```

Comment:
why is this called service account? I'd expect that it'd be called "default-catalog" or "default-url"

also can set the default catalog as default-service-account: "https://github.com/tektoncd/catalog.git"

Reply:
Same. Guilty of copy and pasting from git resolver. Will fix ASAP!